feature/experimental/gribi/ate_tests/route_removal_during_failover_test/route_removal_during_failover_test.go

```
@@ -0,0 +1,565 @@
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/testt"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+
+	spb "github.com/openconfig/gnoi/system"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Settings for configuring the baseline testbed with the test
+// topology.
+//
+// The testbed consists of ate:port1 -> dut:port1
+// and dut:port2 -> ate:port2.
+// There are 64 SubInterfaces between dut:port2
+// and ate:port2
+//
+//   - ate:port1 -> dut:port1 subnet 192.0.2.0/30
+//   - ate:port2 -> dut:port2 64 Sub interfaces:
+//   - ate:port2.0 -> dut:port2.0 VLAN-ID: 0 subnet 198.51.100.0/30
+//   - ate:port2.1 -> dut:port2.1 VLAN-ID: 1 subnet 198.51.100.4/30
+//   - ate:port2.2 -> dut:port2.2 VLAN-ID: 2 subnet 198.51.100.8/30
+//   - ate:port2.i -> dut:port2.i VLAN-ID i subnet 198.51.100.(4*i)/30
+//   - ate:port2.63 -> dut:port2.63 VLAN-ID 63 subnet 198.51.100.252/30
+const (
+	ipv4PrefixLen       = 30              // ipv4PrefixLen is the ATE and DUT interface IP prefix length.
+	IPBlock1            = "198.18.0.1/18" // IPBlock1 represents the ipv4 entries in VRF1
+	nhgID               = 1
+	controlcardType     = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD
+	primaryController   = oc.Platform_ComponentRedundantRole_PRIMARY
+	secondaryController = oc.Platform_ComponentRedundantRole_SECONDARY
+	switchTrigger       = oc.PlatformTypes_ComponentRedundantRoleSwitchoverReasonTrigger_SYSTEM_INITIATED
+	maxSwitchoverTime   = 900
+)
+
+var (
+	dutPort1 = attrs.Attributes{
+		Desc:    "dutPort1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: ipv4PrefixLen,
+	}
+
+	atePort1 = attrs.Attributes{
+		Name:    "atePort1",
+		IPv4:    "192.0.2.2",
+		IPv4Len: ipv4PrefixLen,
+	}
+)
+
+// createIPv4Entries creates IPv4 Entries given the totalCount and starting prefix
+func createIPv4Entries(startIP string) []string {
+
+	_, netCIDR, _ := net.ParseCIDR(startIP)
```

ignoring any error here could come back to bite us. I'd change this function to accept *testing.T as its first argument and to fail fatally if there is a parsecidr error

---

updated, Thank you